src/storage.zig

```diff
@@ -520,6 +520,20 @@ pub fn CreateStorage(comptime all_components: anytype) type {
             }
         }
 
+        /// Get the current number of active entities.
+        ///
+        /// Destorying an entity will add it to the inactive list, hence it is not simply the entity count
```

Comment:
I think we can either remove this comment, or move it above the return statement as it is a implementation detail